app/(dashboard)/mailboxes/[mailbox_slug]/common-issues/commonIssueForm.tsx

```
@@ -0,0 +1,320 @@
+      keywords: commonIssue?.keywords || [],
+    },
+  });
+
+  const keywords = form.watch("keywords") || [];
+
+  const createCommonIssue = api.mailbox.commonIssues.create.useMutation({
+      onSuccess: () => {
+        onSuccess()
+        form.reset()
+      },
+      onError: (error) => {
```

ESLint

[error] Delete `··`

(prettier/prettier)